LobbyCompatibility/Patches/MenuManagerPostfix.cs

```diff
@@ -27,6 +27,9 @@ private static void Postfix(MenuManager __instance)
 
         LobbyCompatibilityPlugin.Logger?.LogInfo("Initializing menu UI.");
 
+        // Set challenge moon leaderboard title text to not wrap halfway through
+        __instance.leaderboardHeaderText.rectTransform.offsetMax = new Vector2(2000, __instance.leaderboardHeaderText.rectTransform.offsetMax.y);
```

Comment:
Just a big number picked randomly, it's extending the textbox on the right so it overflows properly

![image](https://github.com/MaxWasUnavailable/LobbyCompatibility/assets/34404266/7ad1b7ab-0424-4541-b97d-1d3713718636)

Because the textbox is at least two lines high, I don't think there's a "sane" way to make it overflow normally without either:
- Extending the right edge by some big number 
- Shortening the height (and having to deal with the manual repositioning)